Python/Question_20/Validating emial address.md

```diff
@@ -0,0 +1,40 @@
+#Question
```

Comment:
Change the Name of the File. Keep apt underscores and work on the Spell Mistakes